ociinstaller/mediatypes.go

```diff
@@ -52,18 +52,17 @@ const (
 // and it's fallbacks in order of priority
 func MediaTypeForPlatform(imageType ImageType) []string {
 	arch := runtime.GOARCH
-	layerFmt := "application/vnd.turbot.steampipe.%s.%s-%s.layer.v1+tar"
 	switch imageType {
 	case ImageTypeDatabase:
-		return []string{fmt.Sprintf(layerFmt, imageType, runtime.GOOS, arch)}
+		return []string{fmt.Sprintf("application/vnd.turbot.steampipe.%s.%s-%s.layer.v1+tar", imageType, runtime.GOOS, arch)}
```

Comment:
can we define `layerFmtGzip` and  `layerFmtTar`

Reply:
i removed this